### PR TITLE
Updated solo staking section

### LINKED ISSUE #13134: Update solo staking section
## Description
From discussion on #9277:

- [ ] Update terminology to use "home staking" instead of "solo staking"
- [ ] Add informational section about <32 ETH options for home staking
- [ ] Emphasize importance of staking from home; add _brief_ note that liquid staking options are available even for home stakers (need for LST should not stop a user from considering home staking)
- [ ] Add indicator to products that support liquid staking options

These are the key points I've pulled from discussion on the above thread. This will require some help with content writing, and will require design input (potentially with a little design work).

cc: @Jstar101

### PR DESCRIPTION
**Pull request by MukulKolpe**
## Description

Updated terminology from "solo staking" to "home staking." Added info on home staking for users with <32 ETH. Updated links and added StakeWise Operator Service to the Node tools.

For reference, Please review the [Google Doc](https://docs.google.com/document/d/1ALDmRTkd3nO8wgLVTByo6Z8v8cu6M0GWTBhLRRKabHA/edit?usp=sharing) drafted by @Jstar101.

## Related Issue

Ref #9277 

Resolves #13134

### CODE REVIEW

File: public/content/staking/solo/index.md [hunk 1]
```diff
@@ -13,31 +13,31 @@ summaryPoints:
...
 
 <CardGrid>
   <Card title="Earn fresh ETH" emoji="💸" description="Earn ETH-denominated rewards directly from the protocol when your validator is online, without any middlemen taking a cut." />
   <Card title="Full control" emoji="🎛️" description="Keep your own keys. Choose the combination of clients and hardware that allows you to minimize your risk and best contribute to the health and security of the network. Third-party staking services make these decisions for you, and they don't always make the safest choices." />
-  <Card title="Network security" emoji="🔐" description="Solo staking is the most impactful way to stake. By running a validator on your own hardware at home, you strengthen the robustness, decentralization, and security of the Ethereum protocol." />
+  <Card title="Network security" emoji="🔐" description="Home staking is the most impactful way to stake. By running a validator on your own hardware at home, you strengthen the robustness, decentralization, and security of the Ethereum protocol." />
 </CardGrid>
 
-## Considerations before staking solo {#considerations-before-staking-solo}
+## Considerations before staking home {#considerations-before-staking-solo}
```
Comment (Jstar101) [inline public/content/staking/solo/index.md:38]: I think this would read better if it was "Considerations before home staking", rather than "staking home"

File: src/data/staking-products.json [hunk 2]
```diff
@@ -268,6 +309,40 @@
...
+        "discord": "https://discord.com/invite/2BSdr2g",
+        "twitter": "https://twitter.com/stakewise_io",
+        "telegram": "https://t.me/stakewise_io",
+        "github": "https://github.com/stakewise"
+      },
+      "matomo": {
+        "eventCategory": "StakingProductCard",
+        "eventAction": "Clicked",
+        "eventName": "Clicked StakeWise go to link"
+      }
```
Comment (wackerow) [inline src/data/staking-products.json:345]: Let's break these out into a separate PR

**Review by wackerow — CHANGES_REQUESTED** [review]
Hey @MukulKolpe, appreciate your help with this! To keep things a bit more organized, any chance you'd mind breaking the Stakewise updates into a separate PR? The other changes look good, but would prefer to keep these separated.

File: public/content/staking/solo/index.md (hunk 1 above)
Comment (MukulKolpe) [reply public/content/staking/solo/index.md]: Yes, I agree. Updated✅

File: src/data/staking-products.json (hunk 2 above)
Comment (MukulKolpe) [reply src/data/staking-products.json]: Sure, I’ve excluded this part.

**Comment by MukulKolpe** [comment]
Hey @wackerow, sure thing! we can break out the Stakewise updates into a separate PR. Should I create an issue for the same?

**Review by minimalsm — APPROVED** [review]
LGTM now, second pair of eyes @wackerow?
